.travis.yml

```diff
@@ -24,4 +27,4 @@ jobs:
         - taskcat -c ci/taskcat.yml --tag owner=taskcat || travis_terminate 1
     - stage: deploy
       script:
-        - ./deploy-templates.sh || travis_terminate 1
+        - aws s3 cp templates s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/$REPO_NAME/$TRAVIS_BRANCH/ --recursive
```

Comment:
Is this going to copy to a 'templates' folder in the destination?

Reply:
nope, here's a test..
```
➜  aws s3 cp templates s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/aaa/ --recursive
upload: templates/ec2-backup.yaml to s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/aaa/ec2-backup.yaml
upload: templates/bootstrap.yaml to s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/aaa/bootstrap.yaml
upload: templates/GuardDutyMember.yaml to s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/aaa/GuardDutyMember.yaml
..
..

➜  aws s3 ls s3://bootstrap-awss3cloudformationbucket-19qromfd235z9/aaa/
2020-09-15 14:54:01       9424 AccountAlertTopics.yaml
2020-09-15 14:54:01       7147 AddSameRegionBucketDownloadRestriction.yaml
2020-09-15 14:54:01        722 GuardDutyMember.yaml
..
..
```